Forge.TreeWalker/src/TreeWalkerParameters.cs

```diff
@@ -151,11 +162,12 @@ public class TreeWalkerParameters
             this.SessionId = sessionId;
             this.ForgeTree = forgeTree;
             this.ForgeState = forgeState;
-            this.Callbacks = callbacks;
             this.Token = token;
+            this.Callbacks = callbacks;
```

Comment:
Flip the order here. Callbacks before token to line up with parameters.

Reply:
updated

---
In reply to: [506636335](https://github.com/microsoft/Forge/pull/32#discussion_r506636335) [](ancestors = 506636335)